swiftuploader/upload.py

```diff
@@ -12,27 +13,31 @@
 from openstack import profile
 from openstack import utils
 
+CLOUD_CONF = "/home/jenkins/.config/openstack/clouds.yaml"
```

Comment:
We should avoid depending on this too specific path or user - jenkins. May input the path via command line args.